posts/Fabio/2018-07-05-hibernate-ogm-5-4-Beta2-released.adoc

```diff
@@ -0,0 +1,96 @@
+= "Hibernate OGM 5.4.0.Beta2 release"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.Beta2] has been released!
+
+We have several news to announce here. Some of them are:
+
+* We support server side indexes definitions for fast queries with Neo4j.
```

Comment:
Shouldn't this be `server side indexes definition`?

Reply:
Even here is a way to fix the concept that the indexes are stored here within the datastore